chapters/functions.tex

```diff
@@ -2009,7 +2009,7 @@ \subsubsection{Simple Types}\label{simple-types}
 
 An exception is made when the argument is of the form \lstinline!size($\ldots$, $\ldots$)!. In this case the corresponding C type is \lstinline!size_t!.
 
-Strings are \textsc{nul}-terminated (i.e., terminated by \lstinline[language=C]!'\0'!) to facilitate calling of C functions.
+Strings are \textsc{nul}-terminated (i.e., terminated by \lstinline[language=C]!'\0'!) and are encoded using UTF-8 (assuming \lstinline[language=C]!CHAR_BIT==8! in C) to facilitate calling of C functions.
```

Comment:
As the most immediate impact of the choice of encoding is on the MSL string handling functions, it is better to keep this choice open until we have figured out what functionality the MSL string handling functions should provide if `String` values were Unicode, and the proper place for discussing this appears to be https://github.com/modelica/ModelicaStandardLibrary/issues/3789.

Reply:
> I don't think I see what you mean. What duplication? Can we be explicit about which are the other strings that are not of type `String`?

All attributes of StringType, including the start-attribute for String. The duplication is that we have the Unicode-rule twice.

Reply:
I see, so why not simply specify that `StringType` may hold any Unicode characters?

Reply:
Details can now be discussed at the added text instead of here.  Resolving.